dev-docs/modules/permutiveRtdProvider.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: page_v2
+title: Permutive Real Time Data Provider
+display_name: Permutive Real Time Data Module
+description: Permutive Real Time Data Module
+page_type: module
+module_type: rtd
+module_code : permutiveRtdProvider
+enable_download : true
+sidebarType : 1
+---
+
+# Permutive RTD Provider
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+This submodule reads cohorts from Permutive and attaches them as targeting keys to bid requests. Using this module will deliver best targeting results, leveraging Permutive's real-time segmentation and modelling capabilities.
```

Comment:
This is a little awkward. It's referred to as a "submodule" in one sentence, and "module" in the next. Let's stick with module.

What does "deliver best targeting results" mean... better than what? (it's also missing a 'the'?)

Reply:
Yes agreed. I'll remove that sentence. I guess initially it was supposed to say that using the module updates segments more dynamically and therefore enables better targeting results, compared to the alternative route where publishers read segments and add them as parameters to each ad unit. That alternative route is what our clients were doing before we had the module, but it didn't allow to dynamically update the targeting throughout the pageview.